PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -76,6 +77,9 @@ function Invoke-SCuBA {
     Invoke-SCuBA -Version
     This example returns the version of SCuBAGear.
     .Example
+    Invoke-SCuBA -ConfigFilePath MyConfig.json
```

Comment:
Added parameter description.  Rephased a little.  Parameter is not optional if using "Configuration" paramenter set group.  Also currently command line parameters do not override config file.  This will be a future enhancement.  It will require changing the parameter sets.